pkg/ddevapp/dotddev_assets/providers/acquia.yaml.example

```diff
@@ -42,15 +42,14 @@ db_pull_command:
     # set -x   # You can enable bash debugging output by uncommenting
     ls /var/www/html/.ddev >/dev/null # This just refreshes stale NFS if possible
     pushd /var/www/html/.ddev/.downloads >/dev/null
-    acli remote:drush -n ${project_id} -- sql-dump --extra-dump=--no-tablespaces >db.sql
-    gzip db.sql
+    drush @${project_id} sql-dump -n --extra-dump=--no-tablespaces --gzip >db.sql.gz
```

Comment:
Doesnt' this assume you have custom aliases set up for the remote?

Reply:
I suppose it does, but doesn't the acli remote:aliases:download in the auth_command download them?

Reply:
If it's just relying on the ones that acli downloads, it will work fine. If you project has custom aliases, it won't work when they're not there.

Reply:
Actually it must work since the files_pull_command below, line 52/53, is (and was) using the same alias to get the files.

Reply:
I can see one possible issue.   After doing a `ddev pull acquia`, I tried running `ddev drush sa` and got  `[success] No site aliases found.`

Then I tried running `ddev exec acli remote:aliases:download -n` to see the output
```
Drush changed how aliases are defined in Drush 9. Drush 8 aliases are PHP-based, while Drush 9+ aliases are YAML-based.
Cloud Platform Drush aliases installed into /home/gregg/.drush/sites
Drush 9+ does not automatically read aliases from this directory. Run drush core:init to ensure these aliases are discovered.
For more details, see https://github.com/drush-ops/drush/blob/master/examples/example.site.yml
```

So I'm guessing I'll need to add `drush core:init -y` and probably `source ~/.bashrc `so the aliases downloaded show up